tests/ember-intl/tests/integration/helpers/format-date-time-range/input-is-a-date-test.ts

```diff
@@ -42,15 +45,14 @@ module(
 
       await setLocale('de-de');
 
-      assert.dom('[data-test-output]').hasText('23.01.2014 – 26.02.2015');
+      assert
+        .dom('[data-test-output]')
+        .hasText(/23\.–26\.01\.2014/)
+        .hasText('23.–26.01.2014');
     });
 
     test('we can format the date', async function (this: TestContext, assert) {
       await render<TestContext>(hbs`
-        <div data-test-output="1">
-          {{format-date-time-range this.from this.to format='hhmmss'}}
-        </div>
-
```

Comment:
As @miguelcobain noted, [`@formatjs/intl`'s `formatDateTimeRange()`](https://github.com/formatjs/formatjs/blob/%40formatjs/intl%403.0.4/packages/intl/src/dateTime.ts#L118-L123
) doesn't support passing `format` as of `v3.0.4`, like it does with `formatDate()` and `formatTime()`.

To avoid wrong expectations, I decided to not document what would happen if we pass `format`.